dear_petition/petition/etl/tests/test_transform.py

```diff
@@ -57,7 +58,7 @@ def test_combine_batches(batch, batch_file, fake_pdf):
         ciprs_record=record,
         jurisdiction=constants.DISTRICT_COURT,
     )
-    offense_record = OffenseRecordFactory(offense=offense, action="CHARGED")
+    OffenseRecordFactory(offense=offense, action="CHARGED")
```

Comment:
Yeah, and come to think about it a lot of these lines can be removed since this function combines batches using the record data JSON anyway, so the DB objects don't actually matter for this test. I've updated the test accordingly.